### ISSUE TITLE
bug: version1.50.0->version1.51.0 file: textDiffEditor.ts new import function multibyteAwareBtoa  Maximum call stack size exceeded

### ISSUE THREAD

**Issue by YtCsj** [microsoft/vscode#112013]
- VSCode Version: 1.51.0
- OS Version: 

Steps to Reproduce:
![image](https://user-images.githubusercontent.com/19795410/101352569-50347b80-38cd-11eb-827a-72b6a02e96e5.png)
![image](https://user-images.githubusercontent.com/19795410/101352572-5296d580-38cd-11eb-95e1-d33f78a78d09.png)

https://stackoverflow.com/questions/38432611/converting-arraybuffer-to-string-maximum-call-stack-size-exceeded

 ```ts
function toBinary(str: string): string {
	const codeUnits = new Uint16Array(str.length);
	for (let i = 0; i < codeUnits.length; i++) {
		codeUnits[i] = str.charCodeAt(i);
	}
	return String.fromCharCode(...new Uint8Array(codeUnits.buffer));
} 

toBinary(new Array(100000).fill('vs').join('')) // Uncaught RangeError: Maximum call stack size exceeded


```

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by mjbvz** [comment]
Fixed by #112015

**Comment by mjbvz** [comment]
@bpasero We talked through the issue in Redmond and have decided not to fix it in 1.52. Based on my understanding, this should only be triggered for very long file paths and is not a regression compared to 1.51.
